### ISSUE TITLE
NaN in Jupyter and nan in LCA

### ISSUE THREAD

**Issue by datawookie** [dominodatalab/low-code-assistant#29]
Missing values are denoted by `NaN` in Jupyter and by `nan` in LCA. Most people will realise that these are the same thing, however, for consistency it would be much better to have `NaN` in LCA too.

In Jupyter:

![image](https://user-images.githubusercontent.com/6134409/201027769-df9ad47b-3389-4134-ac8a-a00e3cbc9832.png)

In LCA:

![image](https://user-images.githubusercontent.com/6134409/201027800-b3d5d4c8-1f00-4234-b961-e2940bcb9f90.png)

**Comment by maartenbreddels** [comment]
I agree, standard Python formats it this way, but the ultimate authority on this uses 'NaN':
  https://en.wikipedia.org/wiki/NaN
